examples/visualization/styles/helpers/basic-style.html

```diff
@@ -46,11 +46,11 @@
       });
 
       const basicStyle = carto.viz.basicStyle({
-        // color: '#f00',
+        color: '#f00',
         size: 10,
```

Comment:
`size` is not required here (should we create one layer of each type?). 

I would also use lower opacity (eg. 0.5) as the colors you have chosen are quite vivid (and I guess we'll use this as an official example in the future).